internal/mode/mode.go

```diff
@@ -0,0 +1,18 @@
+package mode
+
+import "os"
+
+// Mode return the execution mode
+var Mode string
+
+func init() {
+	Mode = os.Getenv("EXEC_MODE")
+}
+
+// IsTestMode return the execution mode
```

Comment:
add a TODO to remove this and add link to github issue in comments

Reply:
Tracking here https://github.com/razorpay/metro/issues/139